dspace-api/pom.xml

```diff
@@ -632,7 +632,7 @@
         <!-- Used for full-text indexing with Solr -->
         <dependency>
             <groupId>org.apache.tika</groupId>
-            <artifactId>tika-core</artifactId>
+            <artifactId>tika-parsers</artifactId>
```

Comment:
We depend on both tika-parsers *and* tika-core.  For example, `org/dspace/discovery/indexobject/IndexFactoryImpl.java` imports `org.apache.tika.exception.TikaException`.

Reply:
True. I considered having both in the pom, but parsers already brings in core transitively, and the tika docs specifically said:

> If you want to use Tika to parse documents (instead of simply detecting document types, etc.), you'll want to depend on tika-parsers instead

Reply:
It will work as-is.